app/controllers/hiring_staff/base_controller.rb

```diff
@@ -1,32 +1,43 @@
 class HiringStaff::BaseController < ApplicationController
   before_action :authenticate
+  DEFAULT_SCHOOL_URN = '140181'.freeze
```

Comment:
I'm not sure I understand this..  Why do we have hardcoded urns in the app?

Reply:
@despo I've had a crack: https://github.com/dxw/teacher-vacancy-service/pull/76/commits/080a22663ddd3f649a75a85589feb63b5a7b13e2